source/userguide/dm.rst

```diff
@@ -241,6 +241,9 @@ The table below details the ``responder`` methods to send actions, also termed `
 +-------------------------------+----------------------------------------------------------------+
 | :py:meth:`responder.direct`   | Used to send an arbitrary directive object                     |
 +-------------------------------+----------------------------------------------------------------+
+| :py:meth:`responder.sleep`    | Used to put the client to sleep during a specified number of   |
+|                               | milliseconds                                                   |
```

Comment:
*during* or *after*? It would be good clarify this in dialogue.py:52 also, that this `sleep` is dismissing to a standby mode after a delay, not like another meaning of `sleep` like sleeping a process for a duration.

Reply:
I think it should be "after" right?

Reply:
Is this an accurate description: "Dismissing to a standby mode after a delay."

Reply:
I changed the word "during" for after" in dm.rst:244 and added a better description in dialogue.py:52.